E2E tests failing in CI during kind setup

**Describe the bug**
See https://github.com/Azure/orkestra/pull/358/checks?check_run_id=3024269048

```
Fetched 4822 kB in 1s (3579 kB/s)
Reading package lists...
Reading package lists...
Building dependency tree...
Reading state information...
The following NEW packages will be installed:
  helm
0 upgraded, 1 newly installed, 0 to remove and 35 not upgraded.
Need to get 13.7 MB of archives.
After this operation, 45.1 MB of additional disk space will be used.
Get:1 https://baltocdn.com/helm/stable/debian all/main amd64 helm amd64 3.6.2-1 [13.7 MB]
Fetched 13.7 MB in 0s (36.7 MB/s)
Selecting previously unselected package helm.
...
Preparing to unpack .../helm_3.6.2-1_amd64.deb ...
Unpacking helm (3.6.2-1) ...
Setting up helm (3.6.2-1) ...
Processing triggers for man-db (2.9.1-1) ...
Set kubectl context to "kind-orkestra"
Error: an error occurred while uninstalling the release. original install error: timed out waiting for the condition: timed out waiting for the condition
Error: Process completed with exit code 1.
```

--- a/.github/workflows/e2e.yaml
+++ b/.github/workflows/e2e.yaml
@@ -22,11 +22,11 @@ jobs:
           curl https://baltocdn.com/helm/signing.asc | sudo apt-key add -
           sudo apt-get install apt-transport-https --yes
           echo "deb https://baltocdn.com/helm/stable/debian/ all main" | sudo tee /etc/apt/sources.list.d/helm-stable-debian.list
-          curl -s "https://raw.githubusercontent.com/kubernetes-sigs/kustomize/master/hack/install_kustomize.sh"  | bash
+          curl -s "https://raw.githubusercontent.com/kubernetes-sigs/kustomize/master/hack/install_kustomize.sh" | bash
           sudo apt-get update
           sudo apt-get install helm
           kind export kubeconfig --name orkestra
-          helm install orkestra chart/orkestra --atomic -n orkestra --create-namespace --values chart/orkestra/values-ci.yaml
+          helm install orkestra chart/orkestra --atomic -n orkestra --create-namespace --values chart/orkestra/values-ci.yaml --timeout 10m
       - name: Verify Deployment
         run: |
           curl -LO "https://dl.k8s.io/release/$(curl -L -s https://dl.k8s.io/release/stable.txt)/bin/linux/amd64/kubectl"

--- a/.github/workflows/e2e.yaml
+++ b/.github/workflows/e2e.yaml
@@ -11,11 +11,9 @@ jobs:
       - name: Checkout
         uses: actions/checkout@v2
       - name: Create k8s Kind Cluster
-        uses: engineerd/setup-kind@v0.5.0
+        uses: helm/kind-action@v1.2.0
         with:
-          version: "v0.9.0"
-          name: orkestra
-          wait: 10s
+          cluster_name: orkestra
           config: .kind-cluster.yaml
       - name: Deploy
         run: |
@@ -26,7 +24,7 @@ jobs:
           sudo apt-get update
           sudo apt-get install helm
           kind export kubeconfig --name orkestra
-          helm install orkestra chart/orkestra --atomic -n orkestra --create-namespace --values chart/orkestra/values-ci.yaml --timeout 10m
+          helm install orkestra chart/orkestra --atomic -n orkestra --create-namespace --values chart/orkestra/values-ci.yaml
       - name: Verify Deployment
         run: |
           curl -LO "https://dl.k8s.io/release/$(curl -L -s https://dl.k8s.io/release/stable.txt)/bin/linux/amd64/kubectl"

--- a/.github/workflows/e2e.yaml
+++ b/.github/workflows/e2e.yaml
@@ -26,7 +26,7 @@ jobs:
           sudo apt-get update
           sudo apt-get install helm
           kind export kubeconfig --name orkestra
-          helm install orkestra chart/orkestra --wait --atomic -n orkestra --create-namespace --values chart/orkestra/values-ci.yaml
+          helm install orkestra chart/orkestra --atomic -n orkestra --create-namespace --values chart/orkestra/values-ci.yaml
       - name: Verify Deployment
         run: |
           curl -LO "https://dl.k8s.io/release/$(curl -L -s https://dl.k8s.io/release/stable.txt)/bin/linux/amd64/kubectl"

